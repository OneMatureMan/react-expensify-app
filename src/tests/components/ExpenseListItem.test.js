import React from 'react';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';
import { shallow } from 'enzyme';


test('should render ExpenseListItem', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[1]} />);
    expect(wrapper).toMatchSnapshot();
})