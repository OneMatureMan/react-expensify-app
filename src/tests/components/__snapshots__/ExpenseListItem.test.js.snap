// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem 1`] = `
<Link
  className="list-item"
  to="/edit/2"
>
  <div>
    <h3
      className="list-item__title"
    >
      Clay
    </h3>
    <span
      className="list-item__sub-title"
    >
      December 28th, 1969
    </span>
  </div>
  <h3
    className="list-item__data"
  >
    $0.42
  </h3>
</Link>
`;
